mesonbuild/compilers/compilers.py

```diff
@@ -49,7 +49,7 @@
 c_suffixes = lang_suffixes['c'] + ('h',)
 # List of languages that can be linked with C code directly by the linker
 # used in build.py:process_compilers() and build.py:get_dynamic_linker()
-clike_langs = ('d', 'objcpp', 'cpp', 'objc', 'c', 'fortran',)
+clike_langs = ('d', 'objcpp', 'cpp', 'objc', 'c', 'fortran', 'vala',)
```

Comment:
Please open a separate PR for this, it's bad practice to mix commits from unrelated changes together. It also means that your work will get pushed faster.

Please include the sentence "Fixes https://github.com/mesonbuild/meson/issues/791" in your commit message so the issue is closed automatically.

Reply:
Ok, I was planning to do so but I wasn't sure how to get CI to work correctly on this PR. Thanks!